pages/performance/how.md

```diff
@@ -3,6 +3,293 @@ title: How to track performance
 permalink: /performance/how/
 category: Getting started
 layout: styleguide
+lead: A step-by-step guide to picking metrics and tools for tracking performance on your site.
+subnav:
+  - text: Introduction
+    href: '#introduction'
+  - text: Choosing metrics & tools
+    href: '#choosing-metrics-amp-tools'
+  - text: Setting budgets & goals
+    href: '#setting-budgets-amp-goals'
+  - text: Adding site tracking
+    href: '#adding-site-tracking'
 ---
 
-TKTKTK
+## Introduction
+
+Tracking a website’s speed allows a development team to launch and interate on a fast site and makes it easier to incorporate performance into the design and development process. This guide will take you through the steps in tracking performance. It can be used on an existing site or a new site. In this guide, we’ll go over:
+
+{% for subnav in page.subnav %}{% unless subnav.text contains 'Introduction' %}
+1. [{{ subnav.text }}]({{ subnav.href }})
+{% endunless %}{% endfor %}
+
+To make the guide easier to follow, we’ll be taking an actual site, [dashboard.cloud.gov](https://dashboard.fr.cloud.gov), through the entire process. The dashboard is a user interface for the [cloud.gov platform](https://cloud.gov), which allows government agencies to launch servers and sites with ease in the same vein as private products. Throughout this guide, we'll use callouts like the one below to highlight specific examples:
+
+{% include perf_example.html
+  text='Examples will be displayed in a box like this.'
+%}
+
+### What you’ll need
+
+Before starting this guide, it’s important to define some basic information, which will make it easier to choose metrics and tools. If the site is in an early discovery mode, and the code architecture is still unknown, it might be too early to use this guide to track web performance. Tracking performance requires that the technology and architecture for the site is defined. Attempt to define, as best as possible, the following information:
```

Comment:
I looked at previous comments and google drive and can't find any different edits from what this already has